src/main/java/core/basesyntax/converter/StringToFruitDaoConverter.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.dao.FruitDao;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+public class StringToFruitDaoConverter implements Function<String, FruitDao> {
+
+    @Override
+    public FruitDao apply(String s) {
+        String[] arr = s.split(",");
+        if (arr.length != 3) {
+            throw new RuntimeException("Cannot convert string [" + s + "] to fruitDao");
```

Comment:
Consider providing a more informative error message when throwing a `RuntimeException`. This will help in debugging and understanding the exact issue with the input string. For example, specify which part of the string is causing the issue.